plugins/inputs/snmp/snmp.go

```diff
@@ -3,6 +3,7 @@ package snmp
 import (
 	"bufio"
 	"bytes"
+	"errors"
```

Comment:
This import already exists, please do a rebase to current master and add your changes on that, this is creating merge conflicts.